index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap" rel="stylesheet"> 
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+<body>
+  <header>
+    <h1 class="logo">My Logo</h1>
+    <nav>
+      <i class="fa-solid fa-bars fa-2x"></i>
+    </nav>
+</header>
+<main>
+  <h1>I’m Dureti <br>
+    Glad to see <br> you!</h1>
+    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+</main>
+<section>
+  <h3 id="foot">LET’S CONNECT</h3>
+  <div class="socials">
+    <ul>
+      <li><i class="fa-brands fa-twitter"></i></li>
+      <li><i class="fa-brands fa-linkedin-in"></i></li>
+      <li> <i class="fa-duotone fa-m"></i></li>
+      <li><i class="fa-brands fa-github"></i></li>
+      <li><i class="fa-brands fa-angellist"></i></li>
+    </ul>
+  </div>
+</section>
```

Comment:
- Please try to stick as much as possible to the Figma design, which includes:
   - [ ] Increasing the spacing between the header section and the main section to match the Figma design.
   - [ ] Adjusting the background image size to fit the container so that it matches the Figma template.
   - [ ] Using the same font size of the "Let's Connect" text to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/165782115-f822e913-0600-47c2-82b1-7917e32ad30a.png) | ![image](https://user-images.githubusercontent.com/62290545/165782336-75b6bd52-3e1f-4f6b-ba27-c3e3cc8e2271.png) |